administration/tests/test_admin_profiles.py

```diff
@@ -140,13 +140,78 @@ def setUp(self):
         self.profile = AdminProfileFactory()
         self.user = AdminUserFactory()
 
+    def test_Put_Profile(self):
```

Comment:
Change test names to the snake case, please

Reply:
done